### DISCUSSION TITLE
comparison table

### DISCUSSION THREAD

**Discussion by verhulstm** [nebari-dev/nebari#259] [Ideas]
can our docs have something like this?  to explain qhub-cloud versus qhub-onprem?

https://codepen.io/adrianjacob/pen/KdVLXY

**Comment by costrouc** [comment]
@verhulstm I opened and issue to create docs to address this issue. I agree it is important to discuss the reasoning for both and their strengths/weaknesses.

**Comment by Adam-D-Lewis** [comment]
I'm closing old discussions in a repo cleanup effort.  Feel free to re-open if needed.
